fuzz/Cargo.toml

```diff
@@ -34,15 +34,21 @@ name = "instantiate_translated"
 path = "fuzz_targets/instantiate_translated.rs"
 test = false
 doc = false
+required-features = ['binaryen']
 
 [[bin]]
 name = "api_calls"
 path = "fuzz_targets/api_calls.rs"
 test = false
 doc = false
+required-features = ['binaryen']
 
 [[bin]]
 name = "differential"
 path = "fuzz_targets/differential.rs"
 test = false
 doc = false
+required-features = ['binaryen']
+
+[features]
+binaryen = ['wasmtime-fuzzing/binaryen']
```

Comment:
Why don't we force enable the binaryen feature in this crate?

Reply:
Due to the way cargo calculates features it'd actually nullify the benefit of this PR, the feature would still be enabled for `cargo test --all`

Reply:
Can we `--exclude wasmtime-fuzz` so that this crate isn't part of the `cargo test --all` graph?

Reply:
That does work, yeah, but I'd prefer to not require `--exclude` to get things working as expected. It's already sort of a bummer that you have to remember to at least `--exclude lightbeam` and I think it'd be better if we didn't require `--exclude` at all to get a reasonable set of testing locally by default.